dspace-api/src/main/java/org/dspace/statistics/SolrLoggerServiceImpl.java

```diff
@@ -174,6 +177,23 @@ protected SolrLoggerServiceImpl() {
 
     @Override
     public void afterPropertiesSet() throws Exception {
+        statisticsCoreURL = configurationService.getProperty("solr-statistics.server");
+
+        if (null != statisticsCoreURL) {
+            Path statisticsPath = Paths.get(new URI(statisticsCoreURL).getPath());
+            statisticsCoreBase = statisticsPath
+                .getName(statisticsPath.getNameCount() - 1)
+                .toString();
+        } else {
+            log.warn("Unable to find solr-statistics.server parameter in DSpace configuration. This is required for " +
+                     "sharding statistics.");
+            statisticsCoreBase = null;
+        }
+
+        log.info("solr-statistics.server:  {}", statisticsCoreURL);
+        log.info("usage-statistics.dbfile:  {}",
```

Comment:
@nwoodward : I'm curious why we need these `log.info` statements?  It looks like these are both logged already in `SolrStatisticsCore.initSolr()`, so it seems duplicative to log them here again.  Was there a reason you added them here again, or maybe this was just for debugging & it should be removed?

(For me, in the logs, I'm seeing this logged now both from SolrStatisticsCore and then again by SolrLoggerServiceImpl, which is why it looks odd to me.)

Reply:
You're right. I didn't notice them up just a little ways in the logs.